ex9/js/main.js

```diff
@@ -0,0 +1,37 @@
+requirejs(['ajaxGet','load','search', 'order'] , function(ajaxGet, load, search, order) {
+  $(function() {
+    var fun = {
```

Comment:
Again, you don't need this object for calling methods on. If you want this to hold the data, just rename it to `data` (or better, phoneBookData) and access it from the functions as needed.
For the methods that others (order, filter) need to call, you can just pass another object: `return {sortByAsc: ...}`